docs/spec/v1beta3/alerts.md

```diff
@@ -168,9 +177,68 @@ spec:
   inclusionList:
     - ".*succeeded.*"
   eventMetadata:
-    app.kubernetes.io/env: "production"
-    app.kubernetes.io/cluster: "my-cluster"
-    app.kubernetes.io/region: "us-east-1"
+    env: production
+    cluster: my-cluster
+    region: us-east-1
```

Comment:
I'm removing the `app.kubernetes.io/` prefix here since it has absolutely no effect and may only cause confusion in the presence of other more important prefixes the user may need to understand, e.g. `event.toolkit.fluxcd.io/`.